Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Sorry, the remark about git master was a confusion on my side.

So you say that adding `<?xml version="1.0" encoding="UTF-8"?>` and keeping the UTF-8 encoded umlauts does not work? Do you have a simple test scenario so I can see what does not work, maybe a test case?

I'd try hard to avoid adding files with non UTF-8 encoding to the code base, because that makes maintenance more difficult (think of editor settings or spell checkers for example).